django/library/jinja2/library/codebases/releases/retrieve.jinja

```diff
@@ -62,16 +62,16 @@
     <meta property="og:type" content="article">
     <meta property="og:title" content="{{ codebase.title }}">
     <meta property="og:description" content="{{ codebase.summarized_description }}">
-    <meta property="og:image" content="{{ codebase.get_featured_image() }}">
+    <meta property="og:image" content="{{ codebase.get_image_urls()[0] }}">
```

Comment:
do we need to guard against codebases with no images / empty `get_image_urls()` ?

it might be useful to use wagtail's image processing to generate a specifically sized rendition and a URL to that rendition:

https://stackoverflow.com/questions/71509212/getting-url-of-image-on-django-wagtail-site

https://docs.wagtail.org/en/stable/topics/images.html#generating-the-image-as-foo-to-access-individual-properties

https://docs.wagtail.org/en/stable/topics/images.html#full_url